fusillade/directory/authorization.py

```diff
@@ -18,12 +19,18 @@ def get_resource_authz_parameters(user: str, resources: Union[List[str], str]):
     """
     policies = []
     _user = User(user)
```

Comment:
Should this `_user = User(user)` be included in the try/except below? (wondering because of the `User.provision_user` call in the `except` block)

Reply:
No, the user object can be instantiated in python, but the user does not exist in the database until they have been provisioned. User only need to be provisioned into the database once. When a `User` is created in python, fusillade does not check if the users actual exist until a property required the database is requested. This is to save resource by not making request to the database unless needed.